src/ec/packages/ec-component-breadcrumb-standardised/__snapshots__/breadcrumb-standardised.test.js.snap

```diff
@@ -33,14 +33,13 @@ exports[`EC - Breadcrumb Standardised Long renders correctly 1`] = `
       </svg>
     </li>
     <li
-      aria-hidden="false"
       class="ecl-breadcrumb-standardised__segment ecl-breadcrumb-standardised__segment--ellipsis"
-      data-ecl-breadcrumb-standardised-ellipsis="true"
+      data-ecl-breadcrumb-standardised-ellipsis=""
```

Comment:
usual way of rendering things with jest, the one we set a simple attributes names then have this crap. I know it's difficult to keep in mind all these small differences in the rendering we get...

Reply:
Ok, I didn't know that jest was badly rendering attribute without values ;)
I was also referring on this issue: https://github.com/ec-europa/ecl-twig/issues/165#issuecomment-539023572
because sometime we set the `true` value, sometime not.

Reply:
The idea is that we don't set the value for booleans, (aria-hidden is not), so whenever we are hardcoding it in a template i would set it as a name only, sometimes we might have stuff coming from the specs and in that case we print it because otherwise we should start checking whether the value is a boolean, etc...

Reply:
ok, thx for clarification :-)